process-automation/latest/develop/index.md

```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
     * The environment the application is deployed in.
     * When the application should be updated with the new interface.
 
+### Extend task list with custom columns
+
+The Digital Workspace provides a default group of columns for your task list. The columns can be changed and configured in the `process-services-cloud.extension.json` file of the [Process Services Cloud extension library](https://github.com/Alfresco/alfresco-apps/tree/develop/libs/content-ee/process-services-cloud-extension){:target="_blank"}.
+
+Task list with default columns preset.
+
+```json
+{
+  "features": {
+    "taskList": {
+        "presets": {
+            "default": [
+                columns schema
+            ]
+        }
+    }
+  }
+}
+```
+
+![Task list with default columns]({% link process-automation/images/task-list-default-columns.png %})
+
+### Add a column in the task list using a task property
+
+To display the task list with new columns, edit the `process-services-cloud.extension.json` file, and insert an entry into the `features.taskList.presets.default` section.
+
+Task list with new columns.
```

Comment:
I don't think this statement is needed - it's already mentioned in the previous sentence.

Reply:
Changed